src/components/v2/ProgressBar/LabeledProgressBar/index.tsx

```diff
@@ -9,19 +10,21 @@ interface ILabeledProgressBar extends IProgressBarProps {
   whiteLeftText?: string;
   greyRightText?: string;
   whiteRightText?: string;
+  customCss?: SerializedStyles;
```

Comment:
Can we not use `className` like we have so far with other components?

Reply:
Makes sense. I saw you changed the structure of the component and now this is passed through the `className` prop since it's applied to the root div 👍